pkgs/by-name/po/portfolio/package.nix

```diff
@@ -48,7 +48,14 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   dontConfigure = true;
   dontBuild = true;
 
+  # 0.73.0 started shipping bindings for dragonflybsd-x86-64 which is ELFOSABI_SYSV
+  autoPatchelfIgnoreMissingDeps = [
+    "libc.so.8" # from jna plugin dragonflybsd-x86-64/libjnidispatch.so
```

Comment:
Wouldn't it be better to delete the affectd files instead? There is really no reason to ship them to a Linux host.

Reply:
atm we ship all jna files, the dragonflybsd is just new and we break on autopatch try.
Tho i can try to remove them according everything that is not `stdenv.hostPlatform.system`.

Reply:
i removed now manually all jna plugins, that match not our system. I thought initially that i should do some fancy find magic, tho as we just have one supported architecture in `meta.platforms`, i leave that for the situation we add more there.